.changeset/fuzzy-carrots-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/localizations": major
```

Comment:
```suggestion
"@clerk/localizations": minor
```

Reply:
Happy to continue as is, but it has occurred to me that _most_ of this file is identical to `en-US` and it would be a lot easier to maintain if only the changed values were present in this file.

As is, the `en-GB` locale would not automatically benefit from changes to the `en-US` values on the keys where they should be the same.

What do you think? Happy to strip out the identical fields if you agree.

Reply:
@ijxy That’s a good point! Yeah, let’s strip out the duplicates and we’ll merge from there.

Thanks again!